shared/test/integration/helpers/view.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var Marionette = require('marionette-client');
+
+function View(client) {
+    this.client = client.scope({searchTimeout: 5000});
+    this.actions = new Marionette.Actions(this.client);
+}
+
+View.prototype = {
+    getRootElement: function (rootSelector, method) {
+        return this.client.findElement(rootSelector, method);
+    },
+    findElement: function (selectorObject) {
+        return this.client
+        .findElement(selectorObject.locator, selectorObject.by);
+            
+    },
+    rootDisplayed: function () {
+        return this.getRootElement().displayed();
+    },
+    waitForDisplay: function (selectorObject) {
+    	var selector = selectorObject;
+        var self = this;
+        return this.client.waitFor(function () {
+            return self.findElement(selector).displayed();
+        });
+    },
+    waitForHide: function () {
+        var self = this;
+        return this.client.waitFor(function () {
+            return !self.displayed();
+        });
+    },
+    tap: function (selectorObject) {
+        this.actions.tap(this.findElement(selectorObject)).perform();
+    },
+    launch: function (origin) {
+        var client = this.client;
+        client.apps.launch(origin);
+        client.apps.switchToApp(origin);
+
+        // Wait for the document body to know we're really 'launched'.
+        this.client.helper.waitForElement('body');
+    },
+    flick : function(selectorObject) {
+        this.actions.flick(
+        	this.findElement(selectorObject), 0, 0,50, 50, 500).perform();
+        this.actions.wait(2).perform();
+    }
+    ,
+    sendKeys: function(selectorObject,string) {
+        this.findElement(selectorObject).sendKeys(string);
+    },
+    switchToSystemApp: function() {
```

Comment:
:+1: Super nice to have! It'll add more context to the odd `switch_to_frame()` we have everywhere in the Python code.